src/components/utils/dateToStr.js

```diff
@@ -0,0 +1,4 @@
+const dateToStr = dateObj => {
```

Comment:
Widzę, że ta funkcja już istnieje w utils katalog wyżej (chociaż nie robi ona dokładnie tego samego). Jeżeli potrzebujesz obu, to nadaj im bardziej znaczące nazwy, aby było wiadomo do czego służą oraz umieść obie w katalogu utils wyżej. W katalogu components nie powinniśmy trzymać (przy takiej konwencji jak tutaj) funkcji pomocniczych